main_test.go

```diff
@@ -97,13 +97,14 @@ my/package
 			},
 		}
 
-		assert.Equal(t, strings.Trim(`
+		// note: using `$-$` marker to defeat removal of trailing space from `.editorconfig` settings
+		assert.Equal(t, strings.ReplaceAll(strings.Trim(`
 package                                                                            before    after    delta
 -------                                                                           -------  -------  -------
-apples                                                                             32.69%   32.69%   +0.00%
+apples                                                                             32.69%   32.69%      $-$
```

Comment:
This entire line shouldn't exist anymore right?

Reply:
No - the intention was the line would remain - just mute out the delta. I'll add an example.